scripts/src/commands/playwright-docker/Dockerfile

```diff
@@ -13,4 +13,4 @@ RUN npm install -g npm@latest
 RUN npx -y playwright@$PLAYWRIGHT_VERSION install --with-deps
 RUN chmod -R 777 /ms-playwright
 
-CMD npx -y playwright run-server --port ${PORT} --host 0.0.0.0
+CMD npx -y playwright@$PLAYWRIGHT_VERSION run-server --port ${PORT} --host 0.0.0.0
```

Comment:
@geoffgscott I think this is why we keep getting errors since `npx` is pulling latest playwright whenever the tests are run but the browser / install is fixed to a specific version.

Reply:
But also I don't know if we can put this here since it's a docker build arg, does that exist in this context?

Reply:
Pretty sure this can just be 
```
pnpm playwright run-server --port ${PORT} --host 0.0.0.0
```
If not the above is fine.

Reply:
Yeah we might need to install it first though? I'll play with it